packages/Search/src/details/DTK_DetailsDistributedSearchTreeImpl.hpp

```diff
@@ -198,36 +206,96 @@ DistributedSearchTreeImpl<DeviceType>::sendAcrossNetwork(
 
 template <typename DeviceType>
 template <typename Query>
-void DistributedSearchTreeImpl<DeviceType>::deviseStrategy(
-    Point eps, Kokkos::View<Query *, DeviceType> queries,
+void DistributedSearchTreeImpl<DeviceType>::reassessStrategy(
```

Comment:
maybe implement the functions in the order you have declared them. This way it is easier to find them

Reply:
Will swap order in the definitions.